01week/rockPaperScissors.js

```diff
@@ -13,14 +13,45 @@ const rl = readline.createInterface({
 
 // the function that will be called by the unit test below
 const rockPaperScissors = (hand1, hand2) => {
-
   // Write code here
   // Use the unit test to see what is expected
 
```

Comment:
A little more involved than it needed to be, but i like that you reduced everything down to booleans.